osquery/tables/system/posix/xattr_utils.cpp

```diff
@@ -0,0 +1,189 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#ifndef __FreeBSD__
+
+#include <string>
+#include <vector>
+
+#include <sys/xattr.h>
+
+#include <osquery/logger.h>
+
+#include "osquery/tables/system/posix/xattr_utils.h"
+
+#ifdef __APPLE__
+#include <TargetConditionals.h>
+
+#ifndef TARGET_OS_MAC
```

Comment:
What is this gating?

Reply:
This is to make sure that the code is only triggered on a desktop/server macOS operating system

Reply:
Why do you need this here where everywhere else we use __APPLE__?